frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -339,14 +352,149 @@ declare global {
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns/:recurringRunId`,
+        options: { path: { namespace: string; recurringRunId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/create`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/upload`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunsResourceKF | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListExperimentsResponseKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<ExperimentKFv2>,
```

Comment:
There seems to be inconsistencies with the APIs. Shouldn't all pipeline APIs accept a response type of `GoogleRpcStatusKF` as well?

Reply:
@christianvogt , Yeah that make sense , but  referring this https://www.kubeflow.org/docs/components/pipelines/v2/reference/api/kubeflow-pipeline-api-spec/#operation--apis-v2beta1-experiments--experiment_id--get, It seems its missing default response when compared to other api call